# Autoware State Monitor Parameters
/**:
  ros__parameters:
    update_rate: 10.0
    th_arrived_distance_m: 2.0
    th_stopped_time_sec: 1.0
    th_stopped_velocity_mps: 0.01
    wait_time_after_initializing: 1.0
    wait_time_after_planning: 1.0
    wait_time_after_arrived_goal: 2.0